packages/components/src/base-field/types.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import type { ReactNode } from 'react';
+
+export type BaseFieldProps = {
```

Comment:
Looking at the typedefs in the old JavaScript file, I think we're missing `FlexProps`